### DISCUSSION TITLE
Problem with Jupiter Notebook and Texmaker

### DISCUSSION THREAD

**Discussion by mo0915** [MiKTeX/miktex#1495] [Q&A]
I can use the Windows promt to generate the pdf file from a test notepad with code, but Jupiter Notebook and the textmaker can't use Miktex. I have no idea what I can do I use already ChatGPT to find the solution but nothing helped. 

- PATH Environment Variable is correct. 

- already did a new installation of miktex.

**Comment by Dboingue** [comment]
ChatGPT won'T help with configuration delicate problems.   I just fought with it and need to check all its guesses. I would think you might have better luck with google and the blogs it finds.  I think most of the times is about all part using the same python "environment" which is another thing that can go very delicate and confuse the like of Chat GPT.
